docs/UserGuide.md

```diff
@@ -546,23 +541,13 @@ data will display the following result:
   
   <img src="./images/user-guide/FindoExample2.png" alt="FindoExample2">
 
-<<<<<<< HEAD
 * When you enter `findo -d n/Charlotte`, TrackO displays all orders with the name `Charlotte` which have been `delivered`.
 Executing it on our sample data will display the following result:
 
   <img src="./images/user-guide/FindoExample3.png" alt="FindoExample3">
   
 * When you enter `findo -d -p n/Alex` displays all orders with the name `Alex` which have been `paid` and `delivered`.
```

Comment:
I think for these you removed the latest update; yesterday Don asked to add ... `which have been **marked as** delivered` and `which have been **marked as** paid and delivered`, can you help readd them? Thanks Janelle!